rpcs3/Emu/RSX/VK/vkutils/device.h

```diff
@@ -91,6 +95,11 @@ namespace vk
 		const VkPhysicalDeviceMemoryProperties& get_memory_properties() const;
 		const VkPhysicalDeviceLimits& get_limits() const;
 
+#ifdef __APPLE__
```

Comment:
This does not need to be saved. Just load the endpoint on use.
Also avoids the header dependency.